src/Box.php

```diff
@@ -350,19 +350,19 @@ public function sign(string $key, ?string $password): void
     {
         $pubKey = $this->file.'.pubkey';
 
-        Assertion::writeable(dirname($pubKey));
-        Assertion::extensionLoaded('openssl');
+        Assert::writable(dirname($pubKey));
+        Assert::true(extension_loaded('openssl'));
```

Comment:
I think we should specify a message here to help users understand why there is an error